Clarifying installation link. (#566)
void-linux/void-docs: src/installation/guides/fde.md

Before we enter the chroot to finish up configuration, we do the actual install.

> Follow the instructions on [Base Installation](chroot.md#base-installation) to
> install the `base-system` and, additionally, the other important packages we
> need in this specific case, those are: `lvm2` and `cryptsetup`.

Reviewer: ```suggestion
Follow the instructions for a [base installation](chroot.md#base-installation) to
install the `base-system` package together with the packages needed for this guide: `lvm2` and `cryptsetup`.
```

Reviewer: This is missing the mention of GRUB UEFI.

Author: It's because, as I said before, to be more consistent with `chroot.md`, grub could be mentioned only later on as one of the last steps of the installation, and this would help by removing the mentions to modifications on ´/etc/default/grub´ on separate steps of the installation as well, by putting then all together at the end.
This could probably help further improvements of repetition removal on the `fde.md` guide in the future too, since the more consistent the flows of installation are with each other the easier it is to spot duplicated parts.

keys](../../xbps/troubleshooting/common-issues.md#verifying-rsa-keys) for the
packages you are installing.

It should be noted that the next command need to be issued in accordance with
your chosen libc implementation, if musl libc, the link should be directed to:

> `current/musl`, if GNU libc, it should be directed to: `current`.

Reviewer: This isn't complete, aarch64 uses `current/aarch64`

See the `chroot.md` guide, if you could link to it directly from the FDE guide instead of repeating the information here, I think that'd be ideal - ideally this guide would contain only the FDE specific parts, with links to the relevant parts of the chroot one for the rest.

install_items+=" /boot/volume.key /etc/crypttab "
```

Now we will configure our bootloader, see the instructions on [Installing
GRUB](chroot.md#installing-grub) to install the correct GRUB package for your
system, but do not install it on the disk just yet (with `grub-install`) as we

> will need to modify it's configuration before doing so.

Reviewer: I prefer the old language... "Next, install..." just sounds better than "Now we will..."

After installing the correct GRUB package, configure GRUB to be able to unlock
the filesystem. Add the following line to `/etc/default/grub`:

```
GRUB_ENABLE_CRYPTODISK=y
```

Next, the kernel needs to be configured to find the encrypted device. First,
find the UUID of the device.

```
# blkid -o value -s UUID /dev/sda1
135f3c06-26a0-437f-a05e-287b036440a4
```

Edit the `GRUB_CMDLINE_LINUX_DEFAULT=` line in `/etc/default/grub` and add
`rd.lvm.vg=voidvm rd.luks.uuid=<UUID>` to it. Make sure the UUID matches the one
for the `sda1` device found in the output of the
[blkid(8)](https://man.voidlinux.org/blkid.8) command above.

> Next, install the bootloader onto the disk using the correct `grub-install`
> command for your system, as stated on [Installing
> GRUB](chroot.md#installing-grub).

Reviewer: ```suggestion
Next, install the bootloader onto the disk using the correct `grub-install`
command for your system, as explained in the ["Installing
GRUB"](chroot.md#installing-grub) section.
```
